spec/search-api.yaml

```diff
@@ -17,6 +17,8 @@ paths:
       summary: List Tables
       description: Returns a list of Tables.
       operationId: listTables
+      tags:
+        - Tables API
```

Comment:
Are these intended to be standardized in any way?

The service-info API seems to use 'service-info' to describe itself...